hawk/app/models/cib.rb

```diff
@@ -956,6 +956,7 @@ def initialize(id, user, use_file = false, stonithwarning = false)
           # check for guest nodes
           if state == :master || state == :started
             on_node = op.attributes['on_node']
+            next if lrm_resource.attributes['container'].nil?
```

Comment:
IIUC, this will skip the remaining part of the iteration for this `lrm_resource`? If so, may be something like:

```
if lrm_resource.attributes['container'].nil? && (state == :master || state == :started)
...
```
to only let it skip this `if` block?

Don't know why is part is kind of different from master branch.

Reply:
thx yan . we have different branches because SLE12 ,master =SLE15 etc.

yes make sanse to put it inline